service-query/pom.xml

```diff
@@ -22,46 +22,29 @@
                 <spring.boot.container>spring-boot-starter-tomcat</spring.boot.container>
             </properties>
         </profile>
+
         <profile>
             <id>jetty</id>
             <properties>
                 <spring.boot.container>spring-boot-starter-jetty</spring.boot.container>
             </properties>
         </profile>
+
         <profile>
             <id>undertow</id>
             <properties>
                 <spring.boot.container>spring-boot-starter-undertow</spring.boot.container>
             </properties>
         </profile>
-
-        <!-- db -->
-        <profile>
-            <id>mysql</id>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-            </activation>
-            <dependencies>
-                <dependency>
-                    <groupId>mysql</groupId>
-                    <artifactId>mysql-connector-java</artifactId>
-                    <version>6.0.6</version>
-                </dependency>
-            </dependencies>
-        </profile>
     </profiles>
 
     <dependencies>
-        <dependency>
-            <groupId>org.kingsski</groupId>
-            <artifactId>data</artifactId>
-            <version>${project.version}</version>
-        </dependency>
         <dependency>
             <groupId>org.kingsski</groupId>
```

Comment:
Do we need the database dependency? Doesn't it just contain scripts for building the DB?

Reply:
Removed